aws/datadog_policy_macro.yaml

```diff
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: CloudFormation Macro Generates IAM Policy for Datadog Integration Role Dynamically
```

Comment:
```suggestion
Description: CloudFormation macro that dynamically generates the IAM policy for the Datadog integration role
```

Reply:
If there's a reason the description words need to be capitalized feel free to ignore this